src/Conversion/ONNXToTorch/NN/Conv2D.cpp

```diff
@@ -115,9 +116,9 @@ struct ONNXConvOpToTorchLowering : public ConversionPattern {
 
     Value padsList = rewriter.create<PrimListConstructOp>(loc,
         Torch::ListType::get(rewriter.getType<Torch::IntType>()),
-        ValueRange{translatepadsList});
+        ValueRange{translatepadsList.padding});
```

Comment:
This seems to expect `isSymmetric == true` - shouldn't the pass fail if it isn't instead of generating invalid `torch` code?